docs/product-release-notes/src/main/asciidoc/rn-tech-preview-con.adoc

```diff
@@ -33,12 +33,7 @@ see https://access.redhat.com/support/offerings/techpreview/[Technology Preview
 PMML has a partial implementation of the Mining Model type. The Select All, Select First, and Model Chaining sub-types are available.
 
 * Multi-threaded execution: Solves a single data set by partitioning across multiple threads. It is turned off by default and can be enabled either with a KieBaseConfiguration option or with a system property as explained in http://docs.jboss.org/drools/release/7.5.0.Final/drools-docs/html_single/#_multi-threaded_rule_engine[Drools Documentation]
-* Rule Units: Rule Units represent a declarative approach to partitioning a rule set into smaller units, binding different data sources to those units, and the execution of the individual unit. A Rule Unit is comprised of data sources, global variables, and rules.#
-//* Red Hat JBoss Web Server
-
-** You can use the installer to install {PRODUCT} on Red Hat JBoss Web Server with Tomcat 9.
-** You can use Narayana to configure JDBC data sources for {KIE_SERVER}.
-//* Integration with Red Hat JBoss Fuse: Enables you to connect disparate applications, services, or devices in real time.
+* Rule Units: Rule Units represent a declarative approach to partitioning a rule set into smaller units, binding different data sources to those units, and the execution of the individual unit. A Rule Unit is comprised of data sources, global variables, and rules.
```

Comment:
One of the list is not a gerund like the two others - is this intentionsl? OR should be "...and executing the individual unit"? Also "is comprised of" is debatable: https://en.wikipedia.org/wiki/Comprised_of